setup.py

```diff
@@ -10,6 +10,6 @@
     extras_require={
         "dill": ["dill"],
         "hdf5": ["h5py"],
-        "test": ["pytest", "pytest-cov", "ipython"],
+        "test": ["pytest", "pytest-cov", "ipython", "pytest-mpi"],
```

Comment:
Add `"mpi": ["mpi4py"],`

Reply:
Workflows require explicit installation of MPI (currently using openmpi)

Reply:
Yes, it's ok. The MPI option should be there, with a comment in the README that states that a MPI installation is needed for that "extra". See README of https://github.com/Lyncs-API/lyncs.DDalphaAMG